Behavior/BehaviorDelegate.swift

```diff
@@ -17,16 +17,14 @@ import SpeziOnboarding
 import SpeziScheduler
 import SwiftUI
 
-
 class BehaviorDelegate: SpeziAppDelegate {
     override var configuration: Configuration {
+        // swiftlint:disable closure_body_length
```

Comment:
Please remove the SwiftLint disable command here and use the existing computed property below.

Reply:
Please only resolve conversations once you have actually addressed them.